lib/general.sh

```diff
@@ -1067,12 +1067,12 @@ prepare_host()
 
 	display_alert "Build host OS release" "${HOSTRELEASE:-(unknown)}" "info"
 
-	# Ubuntu 20.04.x (Focal) x86_64 is the only fully supported host OS release
+	# Ubuntu 20.04.x (Focal), Ubuntu 21.04.x and Linux Mint 20.x x86_64 are the only fully supported host OS releases
```

Comment:
Hirsute is fully supported only

Reply:
Ah check. You know that btter than me. Thank you for your comment!